api/api.proto

```diff
@@ -1531,7 +1531,17 @@ message SdkVolumeUnmountRequest {
   // MountPath for device
   string mount_path = 2;
   // Options to unmount device
-  map<string, string> options = 3;
+  message Options {
+    // Delete the mount path on the node after unmounting
+    bool delete_mount_path = 1;
```

Comment:
lets update this to `do_not_delete_mount_path` and always delete the mountpath by default.